src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/VisitRepository.kt

```diff
@@ -64,14 +64,15 @@ interface VisitRepository :
 
   @Query(
     """
-    select 
```

Comment:
converted to native SQL since couldn't to the select in a select using JPA query language without and IN clause.